src/shared/components/plots/WaterfallPlot.tsx

```diff
@@ -220,9 +223,13 @@ export default class WaterfallPlot<D extends IBaseWaterfallPlotData> extends Rea
         let max = _(this.waterfallPlotData).map('value').max() || 0;
         let min = _(this.waterfallPlotData).map('value').min() || 0;
 
+        // return range defined by the number of samples for the x-axis,
+        // extend the max domain of y-axis to make up for the offset of x-axis in horizontal mode
+        const order = this.props.horizontal ? [1, this.waterfallPlotData.length] : [1, this.waterfallPlotData.length, this.waterfallPlotData.length+10];
+
```

Comment:
1] Can you also extract the `10` into a constant with a name that provides a clue to what this value does for the plot?

2] I suspect that modification of the plot domain is there to allow for space to shift the plot down in horizontal view. From the docs of victory chart it tells about the`plotDomain`:

>The domain prop describes the range of data the component will include. This prop can be given as a array of the minimum and maximum expected values of the data or as an object that specifies separate arrays for x and y. If this prop is not provided, a domain will be calculated from data, or other available information.

Since we do not need to accomodate new values, I think this approach is not te best way to shift the graph down. Let's discuss when you are back at the office?

Reply:
I think you better use `domainPadding` (https://formidable.com/open-source/victory/docs/common-props/#domainpadding)

Reply:
> I think you better use `domainPadding` (https://formidable.com/open-source/victory/docs/common-props/#domainpadding)

The domain padding is already there but somehow it doesn't work as expected. I will try to figure out why and we will discuss tomorrow.